scripts/ci-install.sh

```diff
@@ -23,10 +23,13 @@ export PATH="$HOME/miniconda/bin:$PATH"
 hash -r
 conda config --set always_yes yes --set changeps1 no
 conda update -q conda
-conda create -y -n test python=3.7
+conda install pip
 conda init
 . $HOME/miniconda/etc/profile.d/conda.sh
-conda activate test
-conda install --file=requirements-$DEV.conda
-pip install -r requirements.txt
-pip install -e . --no-deps
+pip install poetry2conda>=0.3.0
+poetry2conda pyproject.toml --dev -E pytorch -E $DEV -E relay env.yml
+cat $DEV-extras.conda relay-extras.conda >> env.yml
```

Comment:
I suppose that this only works because the yml file ends with the dependency list? This seems like a precarious hack.

Reply:
Yes it is a bit of a hack, but I think it is reliable enough to work.  It should be possible to read the yaml file and modify it with a script if required.

Reply:
Fair enough, this is only temporary until we split the backends into plugin packages.